index.d/dharmit.yml

```diff
@@ -108,3 +108,15 @@ Projects:
     desired-tag: latest
     notify-email: shahdharmit@gmail.com
     depends-on: dharmit/base:latest
+
+  - id: 13
+    app-id: dharmit
```

Comment:
@dharmit can we start adding build-context to new entries from here on out. I will update the old ones.